src/otx/algo/classification/deit_tiny.py

```diff
@@ -18,6 +18,7 @@ def __init__(self, num_classes: int, num_multiclass_heads: int, num_multilabel_c
         config = read_mmconfig(model_name="deit_tiny", subdir_name="hlabel_classification")
         config.head.num_multiclass_heads = num_multiclass_heads
         config.head.num_multilabel_classes = num_multilabel_classes
+        self.image_size = config["data_preprocessor"].get("size", (224, 224))
```

Comment:
Barely image resolution would ever appear in cls mmconfig, moreover, we aim to get rid of them, so, I'd avoid extra mentions of those.

Reply:
Instead, if keeping `self.image_size` in the base class, here we can just assign this value